### DISCUSSION TITLE
Add importer for Google Keep exports

### DISCUSSION THREAD

**Discussion by webster8347** [karakeep-app/karakeep#499] [Ideas]
Add importer for Google Keep exports, c.f. https://support.google.com/keep/answer/10017039?hl=en I would love to replace Keep with Hoarder :-)

(maybe also allow checkboxes, too, as you can do in Keep to create shopping lists or checklists)
![Screenshot 2024-10-06 182046](https://github.com/user-attachments/assets/2a432b7b-7d0b-4237-824f-ac618bb903c3)

**Comment by feickoo** [comment]
I was just about to post this request. I originally use hoarder is because it's likeness to Google Keep and I would like to switch over to Hoarder.

The feature to change note color would be cool too.

And to have url automatic be a link to it.

**Reply by spectre338** [reply]
[https://gist.github.com/spectre338/9b46bb7356cf7d6ad2dce910646b7b26](url)  GitHub Gist

**Reply by crossan007** [reply]
@spectre338 this url is dead now :-(

**Comment by pippo73** [comment]
May I add to the list of users requesting this feature.
Maybe it is possible to use the google-takeout service :-)

**Reply by spectre338** [reply]
I wrote a script to handle this in PowerShell. I just uploaded it to GitHub [https://gist.github.com/spectre338/9b46bb7356cf7d6ad2dce910646b7b26](url) 

The idea is to download Google Keep from Takeout and the process the included JSON into a Hoarder compliant backup file which ca then be imported into you Hoarder instance. Maybe it will help you

**Comment by crossan007** [comment]
A quick search reveals a few "Google Keep Format Converters;" however, none of them directly support Karakeep (or Hoarder):

*  https://github.com/keithfancher/kept 
* https://github.com/amerker/google-keep-converter
* https://help.obsidian.md/import/google-keep

I feel like "native" support for Importing Google Keep (either via Takeout or API) would be a great addition
